lib/ueberauth/strategy/facebook.ex

```diff
@@ -57,6 +58,27 @@ defmodule Ueberauth.Strategy.Facebook do
     end
   end
 
+  @doc """
+  Handles the Facebook callback from mobile.
+  """
+  def handle_mobile_callback(conn, %{token: token}) when is_binary(token) do
```

Comment:
With that cange, we can consider that we could never merge to source repository.